enrolments/validators.py

```diff
@@ -48,6 +48,34 @@ def validate_attendance(classObj):
             )
 
 
+def validate_fields(session_obj):
+    """
+    Validates against the fields column for the Session model.
+    The expected structure is:
+    [
+        {
+            "fields": [
+                // field IDs
+                1, 2, 3, 4, 5
+            ]
+        },
+    ]
```

Comment:
is there a reason why we need to have an outer array? is it valid to have multiple "field" objects this array?

Reply:
now that i take a second look, i don't think is necessary! updating so it's just the JSON object